utils/global-state-update-gen/src/generic/config.rs

```diff
@@ -23,5 +25,27 @@ pub struct Transfer {
 pub struct AccountConfig {
     pub public_key: PublicKey,
     pub balance: Option<U512>,
```

Comment:
What would a `None` mean here?

Reply:
It means that we don't want to change the existing balance. The tool supports changing just some aspects of existing accounts - like, we might want to only remove the stake of a validator, but not touch their account balance.
If the account doesn't exist and this is `None`, it won't be created.